antora-ui-camel/src/css/footer.css

```diff
@@ -182,59 +181,45 @@ input[type="checkbox"] {
     width: auto;
   }
 
-  footer .footer .context a {
-    font-size: 13.5px;
-  }
-
-  footer .footer .context a::after {
-    width: 5px;
-    height: 5px;
-  }
-
   footer .footer .footer-icons .brand-icon {
     height: 1.5rem;
   }
 
-  footer .footer .show-menu {
-    float: right;
-    height: 40px;
-    margin-top: -2.5rem;
-  }
-
-  footer .footer .hide-menu {
-    float: right;
-    height: 35px;
-    margin-top: -2.5rem;
+  footer .footer input[type="checkbox"] {
+    display: inline;
+    position: relative;
+    top: 2rem;
+    left: 45vw;
+    width: 1.75rem;
+    height: 1.75rem;
+    transition: 0.2s;
```

Comment:
In addition to this, there is an outline around the box, to prevent this following code achieves it: 
````
-webkit-appearance: none;
-moz-appearance: none;
-o-appearance: none;
appearance: none;
outline: 0;

Reply:
`appearance` is a [experimental](https://developer.mozilla.org/en-US/docs/Web/CSS/appearance) [feature](https://caniuse.com/#feat=css-appearance)we can't rely on it for website to render on most devices. I've gone with a different implementation in the latest changes.